WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Payment Methods/ShippingLabelPaymentMethods.swift

```diff
@@ -43,9 +51,19 @@ struct ShippingLabelPaymentMethods: View {
         }
         .background(Color(.listBackground))
         .navigationBarTitle(Localization.navigationBarTitle)
-        .navigationBarItems(trailing: Button(action: {}, label: {
-            Text(Localization.doneButton)
-        }))
+        .navigationBarItems(trailing: Button(action: {
+            viewModel.updateShippingLabelAccountSettings {newSettings in
+                onCompletion(newSettings)
+                presentation.wrappedValue.dismiss()
+            }
+        }, label: {
+            if viewModel.isUpdating {
+                ProgressView()
```

Comment:
I just found that the spinner indicator in the navigation bar remains tappable while loading. I think we should disable it while we are in the loading process. Feel free to fix it in this PR or in a new one. 👍

Reply:
Good catch, thanks! Fixed that in 65e1cbc.